recipes/natural_language_processing/chatbot/README.md

```diff
@@ -68,27 +73,72 @@ cd ../recipes/natural_language_processing/chatbot
 _A full list of supported open models is forthcoming._  
 
 
-### Build and Deploy the Model Service
+### Build the Model Service
 
 The complete instructions for building and deploying the Model Service can be found in the
 [llamacpp_python model-service document](../../../model_servers/llamacpp_python/README.md).
 
-The Model Service can be built and ran from make commands from the [llamacpp_python directory](../../../model_servers/llamacpp_python/).
+The Model Service can be built from make commands from the [llamacpp_python directory](../../../model_servers/llamacpp_python/).
 
 ```bash
 # from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
-make -f Makefile build && make -f Makefile run
+make -f Makefile build
+```
+
+or with the podman cmd
+```bash
+podman build -t llamacpp_python . -f  ./base/Containerfile
+``` 
+
+Checkout the [Makefile](../../../model_servers/llamacpp_python/Makefile) to get more details on different options for how to build.
+
+### Deploy the Model Service
+
+The local Model Service relies on a volume mount to the localhost to access the model files. It also employs environment variables to 
+dictate the model used and where its served. You can start your local Model Service using the following `make` or `podman` commands:
+
+```bash
+# from path model_servers/llamacpp_python from repo containers/ai-lab-recipes
+make -f Makefile run
+```
+or 
+```bash
+podman run --rm -it \
+        -p 8001:8001 \
+        -v Local/path/to/locallm/models:/locallm/models \
+        -e MODEL_PATH=models/<model-filename> \
+        -e HOST=0.0.0.0 \
+        -e PORT=8001 \
+        llamacpp_python
 ```
 
-If you wish to run this as a codesnippet instead of a make command checkout the [Makefile](../../../model_servers/llamacpp_python/Makefile) to get a sense of what the code for that would look like.
+### Build the AI Application
 
-### Build and Deploy the AI Application
+The AI Application can be built from the make command:
 
-Make sure the Model Service is up and running before starting this container image. When starting the AI Application container image we need to direct it to the correct `MODEL_ENDPOINT`. This could be any appropriately hosted Model Service (running locally or in the cloud) using an OpenAI compatible API. In our case the Model Service is running inside the Podman machine so we need to provide it with the appropriate address `10.88.0.1`. To build and deploy the AI application use the following:
+```bash
+# Run this from the current directory (path recipes/natural_language_processing/chatbot from repo containers/ai-lab-recipes)
+make -f Makefile build
+```
+or with the podman cmd
+
+```bash
+podman build -t chat . -f  ./app/Containerfile
```

Comment:
This syntax is wrong with the latest layout.

podman build -t chat app/

Would work.

If we document this code in the man page, we have to figure a way to test it to make sure we don't break the docs.